### PR TITLE
Application theming and styling

### LINKED ISSUE #25: should have custom style
Because we are using bootstrap, we need to customize the theme
to have a unique look.

Should have a new font, basic text and background colors used across the app.
Customization for components such as a card, modal, button, input boxes, etc.

### PR DESCRIPTION
**Pull request by kundanvishen**
This PR resolves #25

**Tablet:**
screen shot 2018-07-17 at 6 24 14 am

**Mobile:**
screen shot 2018-07-17 at 6 24 28 am

**Desktop:**
screen shot 2018-07-17 at 6 26 43 am

### CODE REVIEW

File: src/App/App.css [hunk 1]
```diff
@@ -1,17 +1,275 @@
...
+}
+
+.bg-pink {
+  background: #e6a1f2 !important;
+  color: #ffffff;
+  fill: #e6a1f2;
+}
+
+.bg-dark {
+  /* background: #444c67 !important; */
```
Comment (ArfatSalman) [inline src/App/App.css]: No comments!

File: src/App/App.css [hunk 2]
```diff
@@ -1,17 +1,275 @@
...
+}
+
+.bg-warning {
+  background: #ffb64d !important;
+  color: #ffffff;
+  fill: #ffb64d;
+}
+
+.bg-pink {
+  background: #e6a1f2 !important;
```
Comment (ArfatSalman) [inline src/App/App.css]: Why so many !important?

**Review by ArfatSalman — CHANGES_REQUESTED** [review]

**Comment by ArfatSalman** [comment]
Also, it'd be a good idea to start creating abstractions of your component with in-built styles. But move fast too.

**Comment by kundanvishen** [comment]
Would definitely create the abstractions for components from next PR 👍🏻

**Comment by kundanvishen** [comment]
@ArfatSalman Made the changes, please merge.

**Review by ArfatSalman — APPROVED** [review]

**Pull request merged**